scale-tidb-using-tiup.md

```diff
@@ -247,6 +247,14 @@ If you want to remove a TiKV node from the `10.0.1.5` host, take the following s
 >
 > You can take similar steps to remove the TiDB and PD node.
 
+> **Note:**
+>
+> The PD Client in TiKV caches the list of PD nodes. 
+>
+> + In versions before v4.0.3, TiKV cannot automatically and regularly update the cache of the PD node list regularly. Only after the PD leader switches or TiKV restarts to load the latest configuration, the cache can be updated. To avoid the issue of an expired list of PD nodes cached by TiKV, the PD cluster should contain at least one PD node that existed before removing and scaling PD nodes. If this condition is not met, you need to manually perform the PD transfer leader operation to update the PD cache list in TiKV.
```

Comment:
```suggestion
> + In versions earlier than v4.0.3, TiKV does not automatically and regularly update the cache of the PD node list. The cache is updated only after the PD leader switches or TiKV restarts to load the latest configuration. To avoid the issue of an expired list of PD nodes cached by TiKV, the PD cluster should contain at least one PD node that existed before the scaling-in or scaling-out. If this condition is not met, you need to manually perform the PD leader transfer to update the PD cache list in TiKV.
```